### PR TITLE
Add bookmarklet script and link

### PR DESCRIPTION
**Pull request by floriangosse**

### CODE REVIEW

**Comment by KittyGiraudel** [comment]
Hey there, awesome! I’ll check it and merge it. :)

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This small script analyses the heading outline of the document (or given container), and report possible errors. It can be dropped in the DevTools directly, put into a bookmarklet or integrated to a testing suite for example. It’s pretty much up to you.
 
+Bookmarklet: <a href="javascript:!function(e,n,t){n=e.body,t=e.createElement('script'),t.src='https://cdn.rawgit.com/edenspiekermann/outline-audit/master/index.js',t.async=!0,n.appendChild(t)}(document);">Outline</a>
```
Comment (KittyGiraudel) [inline README.md]: This only injects the script itself, but we would need to do `new Outline().warn()` once it’s been executed to actually run the audit.

File: bookmarklet.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+(function (document, body, script) {
```
Comment (KittyGiraudel) [inline bookmarklet.js]: I don’t think it’s necessary to have the uncompressed file in the repo. :)

Comment (floriangosse) [inline bookmarklet.js]: Okay, i will remove it. :)

File: README.md (hunk 1 above)
Comment (floriangosse) [inline README.md]: This is a nice idea. I will add an onload handler which executes `new Outline().warn()`

**Comment by KittyGiraudel** [comment]
Lovely! 💖

**Comment by KittyGiraudel** [comment]
Damnit, GitHub strips these links from README…

**Comment by floriangosse** [comment]
Oh no. I don't check it on my own repo. I only tried a markdown previewer which renders the link correctly.

We could at a simple GitHub page which contains the bookmarklet.

**Comment by KittyGiraudel** [comment]
Check the README, I added a solution. :)

**Comment by floriangosse** [comment]
I saw it to late. The same snippet which will be used for the bookmarklet coud be used as a content snippet in the devtools. Then you will every load and run the current version.

**Pull request merged**